packages/cfd/src/sass/cfd.scss

```diff
@@ -201,12 +201,7 @@
         align-self: baseline;
         margin-inline-start: -0.8rem;
         .dc-text {
-            font-size: var(--text-size-xxs);
             text-align: var(--text-align-left);
```

Comment:
can you also check if we can remove this?

Reply:
Done...